### ISSUE TITLE
Überarbeiten des Wikis für Freifunk Fulda

### ISSUE THREAD

**Issue by mortzel** [freifunk-fulda/orga#49]
Aktuell werden alle Dokumentationen zu Freifunk Fulda im Wiki des Mag.Lab [1] erstellt. Hierbei ist jedoch die Kategorie Freifunk nicht schnell auffindbar. 

Daher sollte entweder ein eigenes Wiki erstellt oder die Menuestruktur des Mag.Lab Wiki ueberarbeitet werden.

[1] https://wiki.mag.lab.sh/wiki/Kategorie:Freifunk

**Comment by SISheogorath** [comment]
ich wäre auch für ein eigenes Wiki, da die CA des Mag.Lab-Wikis nicht default trusted ist, was User abschrecken könnte es sich anzusehen.

**Comment by reissmann** [comment]
Ich bin gegen ein neues Wiki, aber fuer einen austausch des Zertifikates.

Wir betreiben jetzt schon so viel Infrastruktur. Es macht in meinen Augen keinen Sinn alles doppelt und dreifach zu betreiben. Wir haben ganz andere offene Baustellen, als noch ein weiteres Wiki ...

**Comment by mortzel** [comment]
Aufjedenfall sollte die Menuestruktur des Wikis ueberarbeitet werden, da man die Kategorie 'Freifunk' nur ueber die Suchfunktion findet.

**Comment by SISheogorath** [comment]
@mortzel wenn du die erforderlichen Rechte hast, könntest du das direkt übernehmen?

**Comment by reissmann** [comment]
Wenn ihr Euch am Wiki austoben wollt und es an den Rechten haengt, dann einfach Bescheid geben. Ich kann dir Rechte upgraden.

**Comment by reissmann** [comment]
Hmmm, scheinbar kann ich es doch nicht ... 

-> @fooker

**Comment by fooker** [comment]
Hmmmm. Du bist bereits admin. Was genau geht denn nicht?

**Comment by mfuhrmann** [comment]
@mortzel Ich helfe hierbei gerne und hätte auch schon ein paar Ideen dazu.

**Comment by mortzel** [comment]
@mfuhrmann Komme aktuell auf Grund meiner Bachelor Thesis selbst nicht dazu etwas zu machen.

**Comment by mfuhrmann** [comment]
Ja wie? Das ist wichtiger als Freifunk? .... So wird das ja nichts ;)
Dann schreibe ich die Tage mal ein paar Idee hier rein.

**Comment by SISheogorath** [comment]
Ich habe mal die Termine und Angaben auf der Main_Page aktualisiert. hierbei habe ich unter anderem auf den Kalender von Freifunk verwiesen, in dem wir ja vorerst alle anstehenden Termine eingetragen haben. 

Unklar hierbei ist noch, ob unter anderem der Shop-Link ebenso wie der `Bisher erheben wir keine Mitgliedsbeiträge` Teil so stehen bleiben kann. entsprechende Kommentierungen wurden bereits getätigt.

Außerdem wurde der Sitebar ein Link auf die Freifunk Webpage hinzugefügt.

#46 wurde vorerst als "Vorträge und Workshops" hinterlegt, was ich für eine legitime Plattform halte und sinnvoller als über GitHub zu arbeiten, da hier nun jederzeit Anpassungen gemacht werden können ohne dafür einen PR erstellen zu müssen.

Die Seite des [regelmäßigen Treffens](https://wiki.mag.lab.sh/wiki/Regelm%C3%A4%C3%9Figes_Treffen) sollte überarbeitet werden, da hier noch die Bindung an das Café Chaos beschrieben wird. Die Frage die sich mir hier stellt ist, da es vermutlch weiterhin eine solche Bindung geben soll, schreiben wir die Office Factory nun hinzu oder lassen wir das Café Chaos stehen?

Außerdem stellt sich die Frage, wie sinnvoll es ist, die NOP-Seite, in der Navigation zu verlinken, da deren Inhalt "etwas" veraltet ist. 

Um das Ganze etwas abzukürzen, mir würden "Ja/Nein" bzw. "Soll so bleiben/Kann weg" Antworten reichen. insbesondere an @fooker, @mortzel und @reissmann.

**Comment by fooker** [comment]
Termine: Finde ich als Workaround OK. Besser wäre eine Liste der Termine auf der mag.lab Homepage. Schließlich kann und soll es aud den treffen um mehr gehen als um Freifunk.

Den Link zum Shop kann man durchs majors Shop ersetzen. Zumindest verwendet der das aktuelle Logo. Mitgliedsbeiträge müssen noch bis zur Gemeinnützigkeit warten.

Link in der Seitbar und Eintrag unter Vorträge sind OK.

Die Beschreibung der Treffen kann an die OfficeFactory un die Uhrzeit angepasst werden.

Warum ist die Seite der NOP veraltet? Die Konferenz hat nur einfach seit einiger Zeit nicht mehr stattgefunden.

mfg
fooker

## 

-----Original Message-----

**Comment by SISheogorath** [comment]
gut, die Regelmäßiges Treffen Seite wurde angepasst, ebenso wie der Shoplink.

Nun ja, NOP Link, war nur eine Idee. Ich persönlich finde es eher bescheiden, wenn ich Kontent verlinkt habe, der schon X Jahre alt ist, aber das ist meine persönliche Ansicht. Die richtige Reaktion darauf wäre wohl mal wieder die Organisation einer Veranstaltung :D 

Ansonsten würde ich sagen, muss eben, sobald es soweit ist, die inhaltliche Dokumentation wie man was aufzusetzen hat etc. einfach noch in das Wiki aufgenommen werden.

Oder neue Vorschläge genannt, die man umsetzen sollte.

**Comment by reissmann** [comment]
Ja, los gehts, Dustin und ich haben bisher 4 NOPs organisiert und auf jeder auch mindestens einen Vortrag gehalten. Sobald jemand wieder eine organisiert sind wir auch wieder dabei wuerde ich sagen.

Da NOP ein Projekt des Maglab ist, kann das 1) jeder gerne tun und sollte 2) die Webseite verlinkt sein, auch wenn die letzte Veranstaltung ne Weile zurueck liegt.

**Comment by reissmann** [comment]
Das ganze ist wohl eher eingeschlafen? Kommt mir jedenfalls so vor, wenn ich mir z.B. das hier so anschaue: https://wiki.mag.lab.sh/wiki/Freifunk_Fulda

**Comment by fooker** [comment]
Technical details are documented by salt. Everything else in the wiki is outdated. We should get rid of it and move user documentation to the homepage
